GUI/QtQuick/TxListModel.cpp

```diff
@@ -234,53 +230,13 @@ void TxListModel::addRows(const std::vector<bs::TXEntry>& entries)
    }
    if (!newEntries.empty()) {
       QMetaObject::invokeMethod(this, [this, newEntries] {
-         beginInsertRows(QModelIndex(), rowCount(), rowCount() + newEntries.size() - 1);
+         beginResetModel();
          data_.insert(data_.end(), newEntries.cbegin(), newEntries.cend());
-         endInsertRows();
-         emit nbTxChanged();
+         std::sort(data_.begin(), data_.end(), [](auto left, auto right)
```

Comment:
Shouldn't this be performed in external SortFilterModel?
In case you have 1000 TX entries and adding one ZC TX this will result in invalidating all 1001 rows.